### PR TITLE
Feature/word e checker

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i add file word_e_checker.py for practice-3, add module word_e_checker for main.py
#41

### CODE REVIEW

File: python/practices/practice-3/src/path1/word_e_checker.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import os
+
+
+def count_words():
+    """
+    Count and calculate the percentage of words that do not contain 'E' or 'e'
+    in a list of words.
+    """
+
+    words = input("Enter the list of words: ").split()
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/word_e_checker.py]: Words should be passed as a parameter.

File: python/practices/practice-3/src/path1/word_e_checker.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    count_word = len(words)
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/word_e_checker.py]: Update the variable name, such as `total_number_of_words`

File: python/practices/practice-3/src/path1/word_e_checker.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    words_has_no_e = 0
+
+    for word in words:
+        if 'e' not in word and 'E' not in word:
+            print(word)
+            words_has_no_e += 1
+
+    percentage = (words_has_no_e/count_word) * 100
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/word_e_checker.py]: The function name is count_word, it should only do the counting, statistical work should be done in another function.

File: python/practices/practice-3/src/path1/word_e_checker.py (hunk 1 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/word_e_checker.py]: Done

File: python/practices/practice-3/src/path1/word_e_checker.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/word_e_checker.py]: Done

File: python/practices/practice-3/src/path1/word_e_checker.py (hunk 3 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/word_e_checker.py]: Done

File: python/practices/practice-3/src/path1/word_e_checker.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+    for word in words:
+        if 'e' not in word and 'E' not in word:
+            print(word)
+            words_has_no_e += 1
+
+    return words_has_no_e
+
+
+def percentage_words(words):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/word_e_checker.py:28]: Please update the docstring.

File: python/practices/practice-3/src/path1/word_e_checker.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    in a list of words
+
+    Parameters:
+        - words: list of words input
+        - words_has_no_e: list of words not contain letter 'e' or 'E'
+    """
+
+    total_number_of_words = len(words)
+    words_has_no_e = count_words(words)
+    percentage = (words_has_no_e/total_number_of_words) * 100
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/word_e_checker.py]: Have you thought about the case len(words) = 0?

File: python/practices/practice-3/src/path1/word_e_checker.py (hunk 4 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/word_e_checker.py:28]: Done

File: python/practices/practice-3/src/path1/word_e_checker.py (hunk 5 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path1/word_e_checker.py]: Done

File: python/practices/practice-3/src/path1/word_e_checker.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+
+def main():
+    """
+    Count and print percentage word in word list not contains the
+    letter 'e' or 'E'
+    """
+
+    print(os.path.basename(__file__))
+    words = input("Enter the list of words:").split()
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path1/word_e_checker.py:55]: If you have a helper function, please use it.

**Pull request merged**